linux-micro/Dockerfile

```
@@ -0,0 +1,9 @@
+FROM ubuntu:focal
+
+# Install the packages contained in `packages.txt`
+COPY packages.txt /opt/crates-build-env/packages.txt
+RUN apt-get update && \
+    cat /opt/crates-build-env/packages.txt | DEBIAN_FRONTEND=noninteractive xargs apt-get install -y && \
```

```suggestion
    cat /opt/crates-build-env/packages.txt | DEBIAN_FRONTEND=noninteractive xargs apt-get install -y --no-install-recommends && \
```
This saves 20 MB off the final image size.